common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/microsoftsts/MicrosoftStsAuthorizationResponse.java

```diff
@@ -24,6 +24,7 @@
 
 import com.microsoft.identity.common.internal.providers.microsoft.MicrosoftAuthorizationResponse;
 
+import java.net.URL;
```

Comment:
nit: What requires this import?

Reply:
It was left here from previous code by mistake, will remove